libcontainer/process_linux.go

```diff
@@ -364,21 +379,6 @@ func (p *initProcess) start() error {
 						return newSystemErrorWithCause(err, "setting Intel RDT config for ready process")
 					}
 				}
-
-				if p.config.Config.Hooks != nil && len(p.config.Config.Hooks.Prestart) > 0 {
```

Comment:
Removing the pre-start hooks here makes the earlier `// Setup cgroup before prestart hook...` and `// call prestart hooks` comments obsolete.  Now that the pre-start hooks are firing before the start signal (after all creation has finished), maybe we can just drop those comments?